README.md

```diff
@@ -22,7 +22,7 @@ Create a new temporary project:
     `$ cargo-temp rand tokio`
 
 * With a dependency that have a fixed version:
-    `$ cargo-temp anyhow==1.0.13`
+    `$ cargo-temp anyhow=1.0.13`
```

Comment:
That doesn't have a fixed version anymore. https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#comparison-requirements

@yozhgoor maybe you could use the same wording than the doc: "exact version" instead of "fixed version"